### ISSUE TITLE
Anti-tissue transglutaminase antibody positivity 

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6600]
**Preferred term label:** Anti-tissue transglutaminase antibody positivity 

**Synonyms**
anti-tTG or anti-TG2

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
Autoimmune antibody positivity HP:0030057

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**
Celiac disease, OMIM:212750

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
added
